src/transformers/tokenization_bart.py

```diff
@@ -115,6 +115,10 @@ def build_inputs_with_special_tokens(self, token_ids_0, token_ids_1=None) -> Lis
         # We don't expect to process pairs, but leave the pair logic for API consistency
         return token_ids_0 + token_ids_1 + special_tokens
 
+    def set_lang(self, lang: str) -> None:
```

Comment:
hack to allow setting language code and then calling batch_encode_plus for target language.